applications/zpc/components/zwave_command_classes/src/zwave_command_class_meter_control.c

```diff
@@ -244,12 +257,10 @@ static sl_status_t zwave_command_class_meter_supported_get(
     = zwave_command_class_get_version_from_node(node, COMMAND_CLASS_METER);
 
   if (supporting_node_version == 1) {
-    ZW_METER_GET_V5_FRAME *get_frame = (ZW_METER_GET_V5_FRAME *)frame;
+    ZW_METER_GET_FRAME *get_frame    = (ZW_METER_GET_FRAME *)frame;
     get_frame->cmdClass              = COMMAND_CLASS_METER_V6;
```

Comment:
`get_frame->cmdClass              = COMMAND_CLASS_METER;`
maybe change this too for maintainability

Reply:
yes this will not hurt, look previous comments:

https://github.com/SiliconLabs/UnifySDK/pull/12